2.cellprofiler_analysis/scripts/nf1_analysis.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform segmentation and feature extraction for each plate using CellProfiler
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+
+import sys
+sys.path.append("../")
+from utils import cp_utils
+
+
+# ## Set paths for each plate
+
+# In[2]:
+
+
+path_to_pipeline = pathlib.Path("NF1_analysis.cppipe").absolute()
+
+path_to_output = pathlib.Path("analysis_output").absolute()
+
+plates_info_dictionary = {
+    "Plate_1": {
+        "path_to_images": pathlib.Path("../1.cellprofiler_ic/Corrected_Plate_1/").absolute(),
+    },
+    "Plate_2": {
+        "path_to_images": pathlib.Path("../1.cellprofiler_ic/Corrected_Plate_2/").absolute(), 
+    },
+}
```

Comment:
Are nested dictionaries necessary for this data structure? Consider simplifying to increase readability and if reasonable.

Reply:
I only like the dictionary since I can use a for loop with it to run all plates in one cell. If I separate it, wouldn't I then have to have two separate functions?
@d33bs 
This could be just cause I haven't learned of a way to do this differently yet.